dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -125,10 +125,24 @@ CommandBarFlyout::CommandBarFlyout()
             // In that circumstance, we want to have the flyout be open from the start.
             winrt::IFlyoutBase5 thisAsFlyoutBase5 = *this;
 
-            // If we don't have IFlyoutBase5 available, then we assume a standard show mode.
-            if (!thisAsFlyoutBase5 || thisAsFlyoutBase5.ShowMode() == winrt::FlyoutShowMode::Standard)
+            if (auto commandBar = m_commandBar.get())
             {
-                m_commandBar.get().IsOpen(true);
+                // If we don't have IFlyoutBase5 available, then we assume a standard show mode.
+                if (!thisAsFlyoutBase5 || thisAsFlyoutBase5.ShowMode() == winrt::FlyoutShowMode::Standard)
+                {
+                    commandBar.IsOpen(true);
+                }
+
+                // When CommandBarFlyout is in AlwaysOpen state, don't show the overflow button
+                if (AlwaysExpanded())
+                {
+                    commandBar.IsOpen(true);
```

Comment:
>commandBar.IsOpen(true); [](start = 20, length = 24)

Why does AlwaysExpanded affect if we open the commandbar (not the overflow flyout)

Reply:
The CommandBar IsOpen is actually about the overflow flyout! It's unfortunately a bit confusing.